x-pack/plugin/sql/qa/server/src/main/resources/select.sql-spec

```diff
@@ -48,6 +48,8 @@ SELECT salary, first_name, salary AS x, salary y FROM test_emp ORDER BY y LIMIT
 
 constantWithLimit
 SELECT 3 FROM "test_emp" LIMIT 5;
+sameConstantsWithLimit
+SELECT 3, 3, 5 FROM "test_emp" LIMIT 5;
```

Comment:
Work adding a few more variations here:
`SELECT 3, 5, 3`, `SELECT 5, 3, 3, 3` with and without FROM.